spec/views/events/show.html.erb_spec.rb

```diff
@@ -211,6 +211,24 @@
     expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
   end
 
+  it "does not display send acceptances button after acceptances have been sent in selection phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
```

Comment:
I think it is not important if the event is in selection phase. If the acceptances have been sent then the button should not be displayed independent of the current phase. On the other hand I think it is important that there are some open applications for the event. Otherwise the button could be hidden just because there are no (accepted) applications.